src/pyServer/socketServer.py

```diff
@@ -0,0 +1,29 @@
+from flask import Flask
+from flask_socketio import SocketIO
+from transcription import transcribe
+
+app = Flask(__name__)
+socketio = SocketIO(app)
+
+
+@socketio.event
```

Comment:
There seems to be a significant chunk of overlap between this file and app.py. Could we compress them into a single file?

Reply:
Yeah that's why I asked if I should just delete it - it's only really useful for debugging now and isn't in use so I thought should be deleted but wanted to check

Reply:
Fine by me to delete if it doesn't serve any purpose that the other file doesn't

Reply:
Before merging to develop make sure that `socketServer.py` is the file being deleted not `app.py`.

The reason why we want to keep app.py is because the command `flask run` automatically looks for a `app.py`

Reply:
Yep all g I'll delete it now, definitely removing socketServer.py not app.py